.github/workflows/main.yml

```
@@ -20,5 +20,5 @@ jobs:
           - ubuntu-latest
           - windows-latest
         node:
-          - lts/hydrogen
-          - node
+          - 18
+          - latest
```

```suggestion
          - node
```

Not sure about the differences between the two, but `node` seems a bit more stable as a label than `latest`.
Anyway, that’s what it used to be, we could change it but that’s probably a different discussion about trade offs!

---

Yeah, the documentation says that `latest` and `node` are the same, and `latest` sounded clearer to me. But I'm fine with keeping it as it is.